src/components/PopupDelete.js

```diff
@@ -0,0 +1,26 @@
+
+import Popup from "../components/Popup.js";
+import Card from "../components/Card.js";
+export default class PopupDelete extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+  }
+  setConfirmSubmit(handleFormSubmit) {
+    this._handleFormSubmit = handleFormSubmit;
+  }
+  open() {
+    super.open();
+  }
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.